content/updates/2021-12/3-ProcessTracker.md

```diff
@@ -0,0 +1,18 @@
+---
+pim_announcement_img: ./img/Processtracker-filter1.png
+pim_announcement_alt_img: "Process Tracker"
+pim_announcement_audience:
+- EE
+---
+
+# Process Tracker Revamp - Status: Complete!
+::: meta-data type="New" features="Productivity" available="mid-December" in="EE"
+
+Keep track of your tasks and jobs thanks to an improved process tracker section. You can keep a close eye on your jobs much more easily thanks to a brand new progress bar and new filters.
```

Comment:
I tried to recreate this in the PIM but it wasn't showing up for me. Let's add a bit more detail so it's clearer what has been added/changed!